packages/react/src/components/AudioPlayer/AudioPlayer.tsx

```diff
@@ -0,0 +1,258 @@
+  playing: false,
+  volume: 1,
+};
+
+type AudioPlayerAction =
+  | { type: "INITIALIZE"; totalTime: number }
+  | { type: "TOGGLE_PLAYING" }
+  | { type: "UPDATE_TIME"; time: number }
+  | { type: "SEEK_TO"; time: number }
+  | { type: "SET_VOLUME"; volume: number };
+
+const reducer = (state: typeof initialState, action: AudioPlayerAction) => {
+  switch (action.type) {
+    case "INITIALIZE":
+      return { ...state, totalTime: action.totalTime };
+    case "TOGGLE_PLAYING":
+      return { ...state, playing: !state.playing };
+    case "UPDATE_TIME":
+      return { ...state, currentTime: action.time };
+    case "SEEK_TO":
+      return { ...state, currentTime: action.time };
+    case "SET_VOLUME":
+      return { ...state, volume: action.volume };
+    default:
+      return state;
+  }
+};
+
+const getTimeString = (time: number) => {
+  const minutes = Math.floor(time / 60)
+    .toString()
+    .padStart(2, "0");
+  const seconds = Math.floor(time % 60)
+    .toString()
+    .padStart(2, "0");
+  return `${minutes}:${seconds}`;
+};
+
+const getProgressPercentage = (currentTime: number, totalTime: number) => {
+  if (totalTime === 0) {
+    return 0;
+  }
+  return (currentTime / totalTime) * 100;
+};
+
+export const AudioPlayer: React.FC<AudioPlayerProps> = ({
+  className,
+  name,
+  programme,
+  creator,
+  src,
+}) => {
+  const { prefix } = useGlobalSettings();
+  const baseClass = `${prefix}--audio-player`;
+  const audioPlayerClasses = classNames(baseClass, className);
+
+  const [state, dispatch] = useReducer(reducer, initialState);
+
+  const audioRef = useRef<HTMLAudioElement>(null);
+
+  // Example: handle play/pause
+  const togglePlaying = () => {
+    if (audioRef.current) {
+      if (!state.playing) {
+        void audioRef.current.play();
+      } else {
+        audioRef.current.pause();
+      }
+    }
+    dispatch({ type: "TOGGLE_PLAYING" });
+  };
+
+  const handleLoadedMetadata = () => {
+    if (audioRef.current) {
+      dispatch({ type: "INITIALIZE", totalTime: audioRef.current.duration });
+    }
+  };
+
+  const handleTimeUpdate = () => {
+    if (audioRef.current) {
+      dispatch({ type: "UPDATE_TIME", time: audioRef.current.currentTime });
+    }
+  };
+
+  const handleSkipForward = () => {
+    if (audioRef.current) {
+      audioRef.current.currentTime = Math.min(
+        audioRef.current.currentTime + 15,
+        state.totalTime
+      );
+    }
+  };
+
+  const handleSkipBackward = () => {
+    if (audioRef.current) {
+      audioRef.current.currentTime = Math.max(
+        audioRef.current.currentTime - 15,
+        0
+      );
+    }
+  };
+
+  const handleProgressBarClick = (event: React.MouseEvent<HTMLDivElement>) => {
+    if (audioRef.current) {
+      const rect = event.currentTarget.getBoundingClientRect();
+      const clickX = event.clientX - rect.left;
+      const newTime = (clickX / rect.width) * state.totalTime;
+      audioRef.current.currentTime = newTime;
+    }
+  };
```

Comment:
RTL support is missing 

https://github.com/user-attachments/assets/f281fb40-f463-46d4-a477-b33814b31cf7